### PR TITLE
added mean recombination rate

### PR DESCRIPTION
**Pull request by jgallowa07**
I could probably make better tests, any thoughts?

### CODE REVIEW

File: msprime/simulations.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
 import random
 import sys
 import os
+import numpy
```
Comment (jeromekelleher) [inline msprime/simulations.py]: We're using ``numpy as np`` elsewhere, so may as well follow this convention.

File: msprime/simulations.py [hunk 2]
```diff
@@ -804,6 +805,24 @@ def read_hapmap(cls, filename):
             f.close()
         return cls(positions, rates)
```
Comment (jeromekelleher) [inline msprime/simulations.py]: We can make this a property I think, ``mean_recombination_rate``.

File: msprime/simulations.py [hunk 3]
```diff
@@ -804,6 +805,24 @@ def read_hapmap(cls, filename):
...
 
+    def get_mean_recombination_rate(self):
+        """
+        Use numpy to compute the weighted mean recombination rate
```
Comment (jeromekelleher) [inline msprime/simulations.py]: The documentation for a method should tell you *what* it does, not *how* it does it. "Returns the..."

**Comment by jgallowa07** [comment]
@jeromekelleher Awesome, should be all cleaned up now.

**Comment by jeromekelleher** [comment]
Very nice, thanks @jgallowa07!

**Comment by jgallowa07** [comment]
np @jeromekelleher, closing and deleting the branch.

**Pull request merged**